test/gtest/common/test.cc

```diff
@@ -216,6 +221,23 @@ test_base::wrap_errors_logger(const char *file, unsigned line, const char *funct
     return UCS_LOG_FUNC_RC_CONTINUE;
 }
 
+ucs_log_func_rc_t
+test_base::detect_error_logger(const char *file, unsigned line, const char *function,
```

Comment:
need to define this in test_ucp_sockaddr class, since it detects a specific error pattern